### PR TITLE
Scholarshiptracker

### PR DESCRIPTION
**Pull request by AcevedoC17**
Added Delete fetch functionality

### CODE REVIEW

**Comment by AcevedoC17** [comment]
I recommend checking with backend before accepting pull request @AlonsoUriarte14

File: frontend/college-toolbox/src/components/ScholarshipCard.jsx [hunk 1]
```diff
@@ -122,11 +122,31 @@ const ScholarshipCard = ({
...
-    } else {
-      // Perform delete action here or reset state if canceled
-      setResumeFileName('');
-      setIsDeletingResume(false);
     }
+    const formData = new FormData();
+    formData.append('uploadedFile.name', resumeFileName);
+    
+    // Perform delete action here or reset state if canceled
+    fetch('/ScholarshipApplication/delete-resume', {
```
Comment (Poggecci) [inline frontend/college-toolbox/src/components/ScholarshipCard.jsx:130]: Update this fetch with the API_URL
Example found below:
https://github.com/uprm-inso4116-2023-2024-S1/semester-project-college-toolbox/blob/main/frontend/college-toolbox/src/services/authentication.ts

File: frontend/college-toolbox/src/pages/index.astro [hunk 2]
```diff
@@ -1,7 +1,6 @@
 ---
 import LayoutSidebar from '../app/LayoutSidebar.astro';
 import LandingReadme from '../modules/LandingReadme.astro';
-import 'bootstrap/dist/css/bootstrap.min.css';
```
Comment (Poggecci) [inline frontend/college-toolbox/src/pages/index.astro:4]: Update this branch with the latest changes in the main branch
`git merge main`

This line isn't present in the main branch right now

**Review by Poggecci — CHANGES_REQUESTED** [review]

**Review by Poggecci — CHANGES_REQUESTED** [review]
Branch currently has merge conflicts (or at least the web UI indicates this). Once the branch is updated with the main branch it is good to merge👍

**Comment by AcevedoC17** [comment]
Need @AlonsoUriarte14 approval before final merge

**Review by Poggecci — APPROVED** [review]

**Pull request closed without merging**